### DISCUSSION TITLE
vscode如何使 自定义rules 提示 theme/colors 中定义的字段, 类似 bg-black

### DISCUSSION THREAD

**Discussion by LearningUnity** [unocss/unocss#3303] [Q&A] [answered]
![Snipaste_2023-11-03_16-03-09](https://github.com/unocss/unocss/assets/17720234/55be02aa-e26e-4789-9d9c-5a90d701b39b)

**Comment by zyyv** [comment, marked as answer]
Add templates shorthands in your cutstom rules.
https://unocss.dev/config/autocomplete 

```ts
[/^bgi-[xxx]$/, ()=>{}, {
  autocomplete: 'bgi-$colors'
}]
```

**Reply by LearningUnity** [reply]
thank you so much!
